package.json

```diff
@@ -49,7 +49,8 @@
     "eslint": "./node_modules/.bin/eslint magda-web-client/src/",
     "create-secrets": "create-secrets",
     "start-opa": "opa run deploy/helm/internal-charts/opa/policies",
-    "test-opa": "opa test deploy/helm/internal-charts/opa/policies -v"
+    "test-opa": "opa test deploy/helm/internal-charts/opa/policies -v",
+    "cleanup": "rm -rf pancake; for dir in magda-*; do echo \"Removing node_modules from $dir\"; rm -rf $dir/node_modules; done"
```

Comment:
there are two pancake dirs. I guess for purpose of your script you want to delete both?

Reply:
Yep, you are right. Included both.